checkST2110.js

```diff
@@ -671,58 +844,67 @@ const test_20_73_3 = (sdp, params) => {
         continue;
       }
       if (rangePermitted.indexOf(stream.RANGE) < 0) {
-        errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format paramter 'RANGE' is '${stream.RANGE}' and not one of the acceptable values 'FULL', 'FULLPROTECT' or 'NARROW', as per SMPTE ST 2110-20 Section 7.3.`));
+        errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'RANGE' is '${stream.RANGE}' and not one of the acceptable values 'FULL', 'FULLPROTECT' or 'NARROW', as per SMPTE ST 2110-20 Section 7.3.`));
       }
     }
   }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: Test ST2110-20 Section 7.3 Test 3 - RANGE has acceptable values in colorimetry context');
+  }
   return errors;
 };
 
-const maxudpPermitted = [ '1460', '8960' ];
+const maxudpPermitted = ['1460', '8960'];
 
 // Test ST 2110-20 Section 7.3 Test 4 - MAXUDP has acceptable values wrt ST 2110-10
 const test_20_73_4 = (sdp, params) => {
-  let [ mtParams, errors ] = extractMTParams(sdp, params);
-  for ( let stream of mtParams ) {
+  let [mtParams, errors] = extractMTParams(sdp, params);
+  for (let stream of mtParams) {
     if (typeof stream.MAXUDP !== 'undefined') {
       if (maxudpPermitted.indexOf(stream.MAXUDP) < 0) {
         errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'MAXUDP' is '${stream.MAXUDP}' and not one of the acceptable values '1460' or '8960', as per SMPTE ST 2110-20 Section 7.3.`));
       }
     }
   }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: Test ST2110-20 Section 7.3 Test 4 - MAXUDP has acceptable values wrt ST 2110-10');
```

Comment:
I understand `wrt` but if it's going in a message rather than just a code comment, maybe we should spell it out?